stable/airflow/templates/flower/flower-deployment.yaml

```diff
@@ -109,8 +109,8 @@ spec:
                && mkdir -p /home/airflow/.local/bin \
                && export PATH="/home/airflow/.local/bin:$PATH" \
                && echo "*** running flower..." \
-              {{- if .Values.flower.oauth.nabled }}
-               && exec airflow flower --auth={{ .Values.flower.oauth.omains }}
+              {{- if .Values.flower.oauthDomains }}
+               && exec airflow flower --auth="{{ .Values.flower.oauthDomains }}"
```

Comment:
I recomend using `{{ .Values.flower.oauthDomains | quote }}` rather than blindly passing quotes in `bash`